### ISSUE TITLE
Batchnorm update

### ISSUE THREAD

**Issue by swamiviv** [mingyuliutw/CoGAN#3]
Hi, Great way of doing GANs in caffe with the toggle params inside the layer!

I have a minor question: In your scripts, you have specified the learning rates for the batch norm parameters as 1 during training. In traditional caffe-docs, the lr_params for batch norm mean/variance parameters are set to zero since these statistics are accumulated. 

I am wondering why you have used the mean variance parameters as learnable params ?

**Comment by mingyuliutw** [comment]
They are not learnable parameters. By setting the numbers, they do not affect the results. I have recently released a PyTorch implementation of CoGAN via https://github.com/mingyuliutw/CoGAN_PyTorch

Please check the repository for further details.
